tests/conftest.py

```diff
@@ -215,24 +212,11 @@ async def get_access_token(*args, **kwargs):
     access_token_patch.stop()
 
 
-@pytest.fixture(autouse=True)
-def clean_db():
```

Comment:
Where did we move the clean_db logic now?

Reply:
in the `db_session` fixture which now runs in between all the tests